template/contactsearch.hbs

```diff
@@ -1,5 +1,10 @@
 <h3>{{t "contact_search"}}</h3>
+<div class="jsxc-account-choice">
+    <label class="col-sm-4 control-label" for="jsxc-account">{{t "Account"}}</label>
+    <div class="col-sm-8">
+     <select class="form-control" name="account" id="jsxc-account"></select>
+    </div>
+</div>
 <div class="jsxc-content">
     {{t "Loading"}}
 </div>
-<div class="jsxc-results"></div>
```

Comment:
What was the reason to remove the result area? This way the user could change his search without closing, opening the dialog.

Reply:
re readded this... i did not thinkt about that ;)